content/hi/docs/reference/glossary/master.md

```diff
@@ -0,0 +1,15 @@
+---
+title: मास्टर (Master)
+id: master
+date: 2020-04-16
+short_description: >
+  पुराना शब्द, जिसका उपयोग कंट्रोल प्लेन चलाने वाले नोड्स के पर्याय के रूप में किया जाता था।
+
+aka:
+tags:
+- fundamental
+---
+  पुराना शब्द, जिसका उपयोग {{< glossary_tooltip text="कंट्रोल प्लेन" term_id="control-plane" >}} चलाने वाले {{< glossary_tooltip text="नोड्स" term_id="node" >}} के पर्याय के रूप में किया जाता था।
```

Comment:
Use that predefined term  of glossary tooltips in hindi whose translation is already present in our glossary else use default language en

Reply:
Sure, got it.